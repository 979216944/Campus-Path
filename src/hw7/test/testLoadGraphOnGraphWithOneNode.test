# Test load a graph with one node.

# Load graph
LoadGraph oneNode oneNode.tsv

# List nodes
ListNodes oneNode
ListChildren oneNode node1

# Find a nonexist minimum-cost path
FindPath oneNode node1 node2