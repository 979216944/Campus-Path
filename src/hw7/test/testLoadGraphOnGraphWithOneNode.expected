# Test load a graph with one node.

# Load graph
loaded graph oneNode

# List nodes
oneNode contains: node1
the children of node1 in oneNode are:

# Find a nonexist minimum-cost path
unknown character node2